dashboards/Edge-LB/Edge-LB-Go-Runtime.json

```
@@ -53,11 +53,11 @@
   "gnetId": null,
   "graphTooltip": 0,
   "id": null,
-  "iteration": 1563886776152,
+  "iteration": 1564653453289,
   "links": [],
   "panels": [
     {
-      "folderId": 1,
+      "folderId": 0,
```

```suggestion
      "folderId": null,
```
so it searches all folders instead of a static folder; the default folder is created if default dashboards are enabled and subsequent dashboards are uploaded into that folder (should be folderId `1`), otherwise dashboards are uploaded to general (folderId `0`, not technically in a folder). It's safer here to just include all folders in this search so we don't have to make it dynamic in the code (requires code changes to dcos-monitoring) based on whether the default folder is created or not.